app/server/core/layout_manager.py

```
@@ -0,0 +1,81 @@
+    Returns:
+        Dictionary mapping node IDs to Position objects
+    """
+    path = Path(layout_file)
+
+    if not path.exists():
+        return {}
+
+    try:
+        with open(path, "r", encoding="utf-8") as f:
+            data = yaml.safe_load(f)
+    except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)